### PR TITLE
Beny/plt 519 support for settlement claim flow on old contracts

### PR DESCRIPTION
**Pull request by yoshiwarab**
## Description of the change

Updates excess settlement funds claim machine to support older non-shared DAExpSettlement minters

Fixes: PLT-519

### CODE REVIEW

**Review by mattetre — APPROVED** [review]
🙌 💰 

Nice work! Thanks for getting this in so quick 🙏 So for clients (platform) to take advantage of this all that is needed is to bump the SDK version right?

File: packages/sdk/src/machines/utils.ts [hunk 1]
```diff
@@ -20,6 +20,13 @@ export const SUPPORTED_MINTER_TYPES = [
   Minter_Type_Names_Enum.MinterDaExpSettlementV3,
 ];
 
+export const SUPPORTED_SETTLEMENT_CLAIM_MINTER_TYPES = [
+  Minter_Type_Names_Enum.MinterDaExpSettlementV0,
```
Comment (ryley-o) [inline packages/sdk/src/machines/utils.ts:24]: I'm pretty sure we never deployed a V0 minter, but fine to leave here if you want

**Review by ryley-o — APPROVED** [review]
LGTM!

File: packages/sdk/src/machines/utils.ts (hunk 1 above)
Comment (yoshiwarab) [reply packages/sdk/src/machines/utils.ts:24]: Yeah, I just figured I'd include all of them because they have the same ABI for the relevant functions

**Pull request merged**